ui/components/component-library/modal/README.mdx

```diff
@@ -0,0 +1,180 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { Modal } from './modal';
+
+# Modal
+
+The `Modal` is a component that renders a child outside of the normal document flow. It is useful for rendering a child on top of other content.
```

Comment:
Add that it is used together with `ModalOverlay`,`ModalContent` and `ModalHeader`